dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java

```diff
@@ -114,7 +111,7 @@ public void setUrlPattern(String urlPattern) {
 
     public String getEndpointsInfo() {
         final StringBuilder msg = new StringBuilder(1024);
-        final List<EndpointLogLine> endpointLogLines = Lists.newArrayList();
+        final Set<EndpointLogLine> endpointLogLines = Sets.newTreeSet();
```

Comment:
`newTreeSet` accepts a `Comparator`, any reason why you couldn't reuse `EndpointComparator`?